chain/indexer/src/lib.rs

```diff
@@ -50,6 +49,35 @@ impl Indexer {
         Self { actix_runtime: system, view_client, client, near_config }
     }
 
+    /// Initializes genesis and client configs and stores in the given folder
+    /// Exposes `neard::init_configs()`
+    pub fn init_configs(
+        dir: &std::path::Path,
+        chain_id: Option<&str>,
+        account_id: Option<&str>,
+        test_seed: Option<&str>,
+        num_shards: near_primitives::types::NumShards,
+        fast: bool,
+        genesis: Option<&str>,
+        download: bool,
+        download_genesis_url: Option<&str>,
+    ) {
+        if download {
+            openssl_probe::init_ssl_cert_env_vars();
+        }
```

Comment:
I believe we need this fix for `neard init` as well, so there is no point in using this wrapper. Can you check `neard init` on your laptop?